### DISCUSSION TITLE
Apply Google Captcha for some collections?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#1590] [Q&A] [answered]
Hi~
Currently, what's the best way to apply Google Captcha for a collection?

**Comment by ganigeorgiev** [comment, marked as answer]
You'll have to use the [before* request hooks](https://pocketbase.io/docs/event-hooks/#record-api-hooks) and/or define a custom global [middleware/endpoint](https://pocketbase.io/docs/router/).

Captcha support will be eventually added in #731.
